Krimirundgang/TourContent.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+
+namespace Krimirundgang
+{
+    public class Stop
+    {
+        //Stop ID 
+        public int mStopID;
+
+        //Title of the stop
+        public string mTitle;
+
+        //Latitude of the Stop location
+        public double mLat;
+
+        //Longitude of the Stop location
+        public double mLong;
+
+        //Information Text of the Stop
+        public string mText;
+
+        //Preview Text
+        public string mPreview;
+
+        //return Stop ID
+        public int StopID
+        {
+            get { return mStopID; }
+        }
+
+        //return Stop Title
+        public string Title
+        {
+            get { return mTitle; }
+        }
+
+        //return Latitude
+        public double Lat
+        {
+            get { return mLat; }
+        }
+
+        //return Longitude
+        public double Long
+        {
+            get { return mLong; }
+        }
+
+        //return Text of the Stop
+        public string Text
+        {
+            get { return mText; }
+        }
+        //return Text of the Stop
+        public string Preview
+        {
+            get { return mPreview; }
+        }
```

Comment:
This is the simplest form we can use in order to achieve soft "read-only" properties.